elasticdl/python/master/main.py

```diff
@@ -193,6 +194,9 @@ def main():
             "Embedding service start succeeded. The endpoint is %s."
             % str(embedding_service_endpoint)
         )
+        embedding_dims = dict(
```

Comment:
Maybe more this logic to inside `MasterServicer` and pass layers to keep `main.py`'s logic intuitive?

Reply:
`MasterService` already has two arguments that are used by parameter server. They are `init_var` and `embedding_service_endpoint`. `embedding_dims` is also used by parameter server. We can put `embedding_dims` here. Moving these three arguments inside `MasterServicer` can be done in a separate PR.